src/Helpers/APIHelper.js

```diff
@@ -109,7 +109,7 @@ export function getCveListByAccount(apiProps, shouldUseHybridSystemFilter = fals
     return result;
 }
 
-export function getCveIdsList(apiProps) {
+export function getCveIdsList(apiProps, shouldUseHybridSystemFilter = true) {
```

Comment:
Should we remove this hardcoded 'true' and pass `shouldUseHybridSystemFilter` value from the function that is calling `getCveIdsList`?

Reply:
Of course :)